python/sglang/multimodal_gen/envs.py

```diff
@@ -396,6 +398,9 @@ def maybe_convert_int(value: str | None) -> int | None:
             os.getenv("SGLANG_CACHE_DIT_TS_ORDER", "1"),
         )
     ),
+    "SGLANG_USE_RUNAI_MODEL_STREAMER": lambda: get_bool_env_var(
+        os.getenv("SGLANG_USE_RUNAI_MODEL_STREAMER", "true")
```

Comment:
The `os.getenv()` call is not needed.  Otherwise all good👍

```diff
diff --git a/python/sglang/multimodal_gen/envs.py b/python/sglang/multimodal_gen/envs.py
index e855087d7..73d092f8f 100644
--- a/python/sglang/multimodal_gen/envs.py
+++ b/python/sglang/multimodal_gen/envs.py
@@ -399,7 +399,7 @@ environment_variables: dict[str, Callable[[], Any]] = {
         )
     ),
     "SGLANG_USE_RUNAI_MODEL_STREAMER": lambda: get_bool_env_var(
-        os.getenv("SGLANG_USE_RUNAI_MODEL_STREAMER", "true")
+        "SGLANG_USE_RUNAI_MODEL_STREAMER", default="true",
     ),
 }
 
```